### PR TITLE
Document /api/search

### PR DESCRIPTION
**Pull request by mdittmer**

### CODE REVIEW

File: api/README.md [hunk 1]
```diff
@@ -613,3 +614,89 @@ __Example JSON__
...
+`q` value as a substring of the test name will be returned. Defaults to the
+empty string, which will yield all test results for the selected runs.
+
+#### Examples
+
+- https://staging.wpt.fyi/api/search?run_ids=6311104602963968,5132783244541952&q=xyz
+
+__Example JSON__
+
+<details>
```
Comment (mdittmer) [inline api/README.md]: @jugglinmike what am I doing wrong? ( see https://github.com/web-platform-tests/wpt.fyi/blob/748ee3c1e134d69565abc8f0bda661a66feccedc/api/README.md#apisearch )

Comment (jugglinmike) [reply api/README.md]: May be a HTML<->GFM parsing thing. Try separating the tags and the backticks with newlines. Also, don't forget a `<summary>`.

Comment (mdittmer) [reply api/README.md]: Thanks! More newlines seems to fix it.

Comment (jugglinmike) [reply api/README.md]: Newlines are the developer's club soda

**Comment by mdittmer** [comment]
Done. Only put `<details>` around "Examples" for /api/search. Do you want it around all "Examples" sections? In this PR?

**Comment by mdittmer** [comment]
So, to be clear, my most recent change put a very short list of `Examples` into a `<details>` element, *in addition to* the long `Example JSON`. Other sections have `Example JSON` and *not* `Examples` inside a `<details>` element. Should I revert the extra `<details>` around the one example link to make it consistent with others?

**Comment by Hexcles** [comment]
I see. No I don't think it matters unless it bothers you :)

**Pull request merged**